.github/workflows/sycl_windows_build_and_test.yml

```diff
@@ -0,0 +1,68 @@
+name: SYCL Windows Test
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-2022
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: src
+        fetch-depth: 1
+    - name: Install dependencies
+      shell: cmd
+      run: |
+        choco install -y cuda --version 11.6.0.51123
+        choco install -y ninja
+        choco install -y sccache --version 0.2.15
```

Comment:
In general looks good to me, although there are unnecessary steps in the script due to disabled cache.

Reply:
This comment is not just about installing unused software, but also other parts of the script (e.g. configuring sccache). See lines 26-34 and 45-47. Are you going to remove them as well?

Reply:
@bader I would prefer them to stay here and I will create trackers to fix runtime CMakes and to re-enable sccache in testing

Reply:
Sounds good to me. Thanks.